src/pages/Landing/Phases/index.tsx

```diff
@@ -55,7 +55,7 @@ const phases: phase[] = [
       'The duration for the deployment of the individual phases depends on many factors.',
     linkUrl:
       'https://docs.ethhub.io/ethereum-roadmap/ethereum-2.0/eth-2.0-phases/',
```

Comment:
Should we use https://ethereum.org/en/eth2/#roadmap, which includes Phase 1.5?
/cc @djrtwo @jjmstark

Reply:
I agree with this idea. I've updated all of the phase descriptions to better explain phase 1.5 and the link now points to the eth2 roadmap instead of the ethhub docs.